src/BalconyBotanica/wwwroot/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+
+    <h1>Find your balcony plant</h1>
+
+
+    <form id="plantForm">
+        <h2><label for="sunlight">&#x27A1; Amount of sunlight</label></h2>
+        <input type="radio" id="sunlight" name="sunlight" value="FULL_SUN">Full sun <br />
+        <input type="radio" id="sunlight" name="sunlight" value="PART_SHADE">Part shade<br />
+        <input type="radio" id="sunlight" name="sunlight" value="FILTERED_SHADE">Filtered shade<br />
+        <input type="radio" id="sunlight" name="sunlight" value="FULL_SHADE">Full shade<br />
+
+        <h2><label for="spaceSize">&#x27A1; Space size</label></h2>
+        <input type="number" id="spaceSize" name="spaceSizeSquareMeters" value="m2">m2<br />
+
+        <h2><label for="watering">&#x27A1; Watering</label></h2>
+        <input type="radio" id="watering" name="wateringSchedule" value="MINIMUM">MINIMUM<br />
+        <input type="radio" id="watering" name="wateringSchedule" value="AVERAGE">AVERAGE<br />
+        <input type="radio" id="watering" name="wateringSchedule" value="FREQUENT">FREQUENT<br />
+
+        <h2><label for="function">&#x27A1; Plant function</label></h2>
+        <input type="radio" id="function" name="plantFunction" value="EATABLE">EATABLE<br />
+        <input type="radio" id="function" name="plantFunction" value="FLOWER">FLOWER<br />
+        <input type="radio" id="function" name="plantFunction" value="GREEN">GREEN<br />
+        <input type="radio" id="function" name="plantFunction" value="INSECTS">INSECTS<br />
+        <input type="radio" id="function" name="plantFunction" value="INSECTS">NONE<br />
+
+
+        <h2><label for="toxicity">&#x27A1; Toxicity</label></h2>
+        <input type="radio" id="toxicity" name="toxicity" value="POISONOUS_TO_HUMANS">POISONOUS_TO_HUMANS<br />
```

Comment:
This is interesting. In the database, we probably want to store that a plant is poisonous to humans and/or pets. But in the UI, the user probably wants to select NOT poisonous to humans/pets. Are you able to change the UI and then invert the selection to find the right plant? That would be an interesting exercise

Reply:
ah yess, good catch!